### ISSUE TITLE
add language attribute to html tag of index.html

### ISSUE THREAD

**Issue by noopkat** [noopkat/electric-io#112]
Related to the accessibility audit that @escherina ran via issue #1 

The following [index.html](https://github.com/noopkat/electric-io/blob/master/public/index.html#L2) file is missing the `lang` language attribute. Adding it with the value of `en` (shorthand for English) will fix one of our accessibility audit failures.

If you'd like to take this on, comment on this issue, and go right ahead with the changes 😄 
If you need to look it up, the [contributing guide](https://github.com/noopkat/electric-io/blob/master/CONTRIBUTING.md) is in the root directory of this repository.

**Comment by FuzzyPersian** [comment]
@noopkat Would love to jump on this. I'm super brand new so just give me a bit but I'm excited to work on this. 😄

**Comment by kleinfreund** [comment]
Take all the time you want/need, @FuzzyPersian. No rush. :)

**Comment by FuzzyPersian** [comment]
@kleinfreund @noopkat 
Can I push my issue/112 branch up to this remote for you guys to review?
also not sure why I got this message message on commit, I only did 1 addition.  hopefully I didn't mess anything up. 
 1 file changed, 1 insertion(+), 1 deletion(-)

**Comment by kleinfreund** [comment]
The easiest way to review a pull request for a project maintainer is by submitting a pull request from a copy of the remote repository (usually called “fork”). In most cases, you won’t have the permission to push your changes directly to the project when contributing to open source software.

You most likely cloned this repository, right? If so, I’m going to ask you to also fork it (there is a fork button at the top of this page, for example). This creates yet another clone of this repository. Now there exist three versions of this repository: The original, the one you cloned locally to your computer, and another clone, but this one is also online like the original. From these online copies (forks), you can create pull requests if you pushed commits to it.

For this, you need to tell your local repository that there exists another remote version. Currently, it only knows the original (which it knows under the name “origin”).

```sh
git add remote fork https://github.com/FuzzyPersian/electric-io.git
```

Now, this is how you push your changes from a specific branch to a specific remote repository:

```sh
git push fork issue/112
```

Once this is done, you can go to your fork on GitHub. It should now ask you right away if you want to create a pull request. Clicking that button should set you up with a text field just like when creating a new issue on GitHub. Fill it out and submit the pull request. Then, we can review it here. It will pop up under “Pull requests”. In this repository.

Now, I know this might be overwhelming – apologize if you know these things already –, but hang in there. GitHub and git are not trivial tools to understand and take some time getting used to. Hope that helps. :)

**Comment by FuzzyPersian** [comment]
You are awesome. 

You should never apologize and you should ALWAYS assume that I don't know this stuff, I'm still in learning mode and your explanation was fantastic.  Love learning this stuff. 

Thank you!

done and hopefully I did a decent job, other than my spelling mistakes in the commit comments.

**Comment by noopkat** [comment]
This thread is so wholesome that it's actually a sad thing to close it now that the issue is solved via #114 
Thanks everyone for being awesome 💖

**Comment by nandub** [comment]
Can we made kleinfreund explanation a wiki page? Just so that new people that wants to contribute know how. My 2 cents.

**Comment by noopkat** [comment]
@nandub this is a great idea! I think perhaps it would be most helpful to be placed in the Contributing Guide under the [Pull Request section](https://github.com/noopkat/electric-io/blob/master/CONTRIBUTING.md#pull-requests), what do you think?

**Comment by nandub** [comment]
@noopkat yes, agree.  That’s a great place to put it.
